src/traceroute.ts

```diff
@@ -0,0 +1,130 @@
+const completionSpec: Fig.Spec = {
+  name: "traceroute",
+  description: "Print the route packets take to network host",
+  options: [
+    {
+      name: ["--help", "-h"],
+      description: "Show help for traceroute",
+    },
+    {
+      name: "-a",
+      description: "Turn on AS# lookups for each hop encountered",
+    },
+    {
+      name: ["-A", "as_server"],
```

Comment:
I think `as_server` actually is the argument of the option and not another way of calling it.
```suggestion
      name: "-A",
      args: { name: "as_server" },
```
It's the same for other options of this spec, could you fix those?

Reply:
Hi @fedeci thanks for the review, just updated the arguments.